pde_superresolution/weno_test.py

```diff
@@ -28,21 +28,24 @@
 class WENOTest(parameterized.TestCase):
 
   def test_calculate_omega_smooth(self):
-    u = np.zeros(5)
+    u = np.linspace(0, 1)
     actual = weno.calculate_omega(u)
-    expected = np.stack(5 * [[0.1, 0.6, 0.3]], axis=1)
+    actual = actual[:, 2:-2]
```

Comment:
can you add a brief comment noting why we're indexing here? (to trim out points near the discontinuity)

also, for consistency we might always do `actual[:, 2:-3]` (see my comment below).